src/interfaces.ts

```diff
@@ -514,9 +613,29 @@ export interface PlaybackProgressUpdatedEvent extends Progress {
   track: number;
 }
 
-export interface PlaybackStateEvent {
-  state: State;
-}
+export type SleepTimer =
+| { time: number }
+| { sleepWhenPlayedToEnd: boolean }
+| null
+
+export type SleepTimerChangedEvent = SleepTimer;
+
+export type PlaybackState =
+  | {
+      state:
+        | State.None
```

Comment:
@puckey is there a reason that `State.Ended` is not included in this list? My assumption is that you would either get an error state, or a state with any of the other possible values.

If it's an oversight I can add it to my open PR.

Reply:
See https://github.com/doublesymmetry/react-native-track-player/pull/1713/commits/9e9c2caeb88e60ad202224d86533da470391387b